OWNERS

```diff
@@ -1,10 +1,9 @@
 approvers:
 - evankanderson
 - grantr
-- inlined
-- n3wscott
-- scothis
+- Harwayne
```

Comment:
this diff looks odd because it looks like you're adding Harwayne (who's already in the head as well as removing inlined and scothis)

Reply:
not sure what view I was looking at...